setup.py

```diff
@@ -53,7 +53,7 @@ def readme():
     os.path.dirname(get_python_inc()),
     get_python_inc()
 ]
-library_dirs = [get_config_var("LIBDIR")]
+library_dirs = [get_config_var("LIBDIR")] if os.name == "posix" else []
 sources = glob("src/*.pxd") + glob("src/*.pyx") + glob("src/*.cpp")
 libraries = ["boost_container"]
```

Comment:
This name needs to be changed for Windows. Unfortunately Boost on Windows hardcodes the Boost version and the VC version in the name. 😖 So we would need to get this info somehow or guess. Sadly we will probably have to do both. 😭